src/ProcessService/ProcessServiceUtils.cpp

```diff
@@ -191,9 +191,17 @@ std::optional<TotalCpuTime> GetCumulativeTotalCpuTime() {
 
   std::vector<std::string_view> splits = absl::StrSplit(first_line, ' ', absl::SkipWhitespace{});
 
+  auto begin_field = splits.begin() + 1;  // The first field is the cpu ID `cpuXX`, so we skip that.
+  auto end_field = std::min(begin_field + 8, splits.end());
+  // We accumulate up to 8 fields. (Depending on what kernel version is available there might be
+  // fewer or more fields. We skip the 9th (`guest`) and the 10th field (`guest_nice`) because they
+  // are already included in the first field (`usertime`).)
+
+  if (begin_field >= end_field) return std::nullopt;
+
   const Jiffies jiffies{
-      std::accumulate(splits.begin() + 1, splits.end(), 0ul, [](auto sum, const auto& str) {
-        int potential_time = 0;
```

Comment:
This 32bit integer was the main culprit that led to #4333. Some of the fields might not fit into an `int`.

The skipping of the `guest` and `guest_nice` fields above make the percentage even more correct but they don't have a big impact.

Reply:
Ha, thanks the explanation!